internal/terminal/table.go

```diff
@@ -131,6 +131,12 @@ func parseValue(value interface{}) string {
 	case nil: // leave zero-value
 	case string:
 		parsed = v
+	case bool:
+		if value == true {
```

Comment:
[Q] do we need to do the equate? Could it just be 
`if value {`

Reply:
`value` is still seen as an `interface{}`, but you should be able to use the reference to `v` for that:

```go
if v { parsed = "yes" } else { parsed = "no" }